packages/uicore/src/components/MultiTypeInput/MultiTypeInput.tsx

```diff
@@ -84,6 +85,11 @@ export const getMultiTypeFromValue = (
     return MultiTypeInputType.FIXED
   } else if (typeof value === 'string') {
     value = value.toLocaleLowerCase().trim()
+
+    if (value.startsWith(RUNTIME_INPUT_Y1_PREFIX)) {
+      return MultiTypeInputType.RUNTIMEY1
+    }
```

Comment:
We need to check if we have expressions that starts with `<+inputs.`. If this is a case in v0 it will show v1 runtime input

Reply:
Confirmed with BE that values starting with `<+inputs.` are always runtime inputs.